### DISCUSSION TITLE
Please tell me how "pie and donut charts" can solve the problem on my iPhone.

### DISCUSSION THREAD

**Discussion by kimdonghyun1219** [cloudscape-design/components#1101] [Q&A] [answered]
https://cloudscape.design/components/pie-chart/?tabId=playground

On the iPhone, the configuration properties were checked "hideDescriptions, hideTits" to show only the pie chart.
But this is where the problem arises.
The web browser window goes to the top when you touch right next to the chart boundary or close the description pop-up. I think "reflow" is happening, is there any solution?

**Comment by rubencarvalho** [comment]
Hello,

I can confirm I could reproduce it in our playground. We will investigate the issue further and post our updates here.

Thank you for reporting.

**Comment by timogasda** [comment, marked as answer]
I just merged some code that should fix the bug: https://github.com/cloudscape-design/components/pull/1130. It might take another day or so until this makes it into npm, though.
